<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employe</class>
 <widget class="QDialog" name="employe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>1151</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Ajouter employé</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_nom">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>160</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prenom">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>230</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_nom_ajouter">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>170</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_fonction_ajouter">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>370</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Fonction</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_fonction">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>370</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_matricule_ajouter">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>100</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Matricule</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>450</y>
       <width>141</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_salaire_ajouter">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>300</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Salaire</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_matricule">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>100</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_salaire">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>300</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_prenom_ajouter">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>230</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Prénom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="annuler">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>450</y>
       <width>141</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Afficher liste des employés</string>
    </attribute>
    <widget class="QPushButton" name="PDF">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>520</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Générer PDF</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>591</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Fonction</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rec">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>280</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>390</y>
        <width>141</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter Employé</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>290</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>290</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirmer Mot de passe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="tab_emp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>741</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>420</y>
       <width>296</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>454</y>
       <width>296</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_tri_prenom">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>300</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri croissant prenom</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_tri_salaire">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>300</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri croissant salaire</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_tri_fonction">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>300</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri croissant fonction</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_tri_nom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri croissant nom</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>50</y>
       <width>139</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="affichage_matricule">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="affichage_nom"/>
      </item>
      <item>
       <widget class="QLineEdit" name="affichage_prenom"/>
      </item>
      <item>
       <widget class="QLineEdit" name="affichage_salaire"/>
      </item>
      <item>
       <widget class="QDateEdit" name="affichage_date">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="affichage_fonction"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>60</y>
       <width>104</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_matricule_rec_4">
        <property name="text">
         <string>Matricule</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_nom_rec_4">
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_prenom_rec_4">
        <property name="text">
         <string>Prénom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_salaire_rec_4">
        <property name="text">
         <string>Salaire</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_dateE_rec_4">
        <property name="text">
         <string>Date D'embauche</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_fonction_rec_4">
        <property name="text">
         <string>Fonction</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Chat</string>
    </attribute>
    <widget class="QPushButton" name="lancerChat">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>260</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Lancer le Chat</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>20</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
